.github/workflows/client.yml

```diff
@@ -0,0 +1,42 @@
+name: Go
+
+#TODO: extend the conditions once workflow gets tested together with other workflows 
+on:  
+  push:
+    branches:
+      - master
```

Comment:
One thing that may be worth considering here, if we're continuing to run the Circle build, is to change this filter to run on certain branches (e.g. `rfc-18/*`) and change Circle to _not_ run on that filter. This effectively gives you a branch namespace to use for the work.

Reply:
Implemented in https://github.com/keep-network/keep-core/pull/2329